.asset-template.json.hb

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Block Coding",
+  "description": "Create games using a high-level, block-based visual programming language.\r\n\r\nIntended as an educational tool for learners in the earlier stages of their journey towards becoming game developers. This plugin lets you create your first games with high-level blocks, avoiding the immediate need to learn to code in GDScript. Building games in this way provides a gentle introduction to programming concepts and allows you to focus your efforts on becoming familiar with the rest of the Godot Editor UI.",
+  "category_id": "5",
+  "godot_version": "4.2",
+  "version_string": "{{ context.release.tag_name }}",
+  "cost": "MIT",
+  "support_level": "community",
+  "download_provider": "GitHub",
+  "download_commit": "{{ env.GITHUB_SHA }}",
```

Comment:
Looking at the [assetlib API reference](https://github.com/godotengine/godot-asset-library/blob/master/API.md#post-asset-post-assetid-post-asseteditid), I believe you can use a ref here. In which case, I'd rather have `{{ context.release.tag_name }}` than a raw hash. Not sure about that, though.

Reply:
We can try.

One property a commit hash has that a tag doesn't is that we can't change what it points to after the fact. But that example pointing directly to `master` suggests that the assetlib maintainers don't really mind that!

I tried requesting a few arbitrary asset IDs and eventually I did find [one](https://godotengine.org/asset-library/api/asset/266) that uses a tag name here.

Reply:
Switched this to the tag name.

Reply:
Tag names are no longer accepted. I pushed a fixup to switch this back to using the SHA.

https://github.com/endlessm/godot-block-coding/commit/3daad38da895fe3b0a4ba4ad410be05500e90c8a